src/lib/plugin_apis/lvm.api

```diff
@@ -5,13 +5,13 @@
 #ifndef BD_LVM_API
 #define BD_LVM_API
 
-#ifdef __LP64__
-// 64bit system
-#define BD_LVM_MAX_LV_SIZE (8 EiB)
-#else
-// 32bit system
-#define BD_LVM_MAX_LV_SIZE (16 TiB)
-#endif
+/**
+ * BD_LVM_MAX_LV_SIZE:
+ *
+ * Deprecated: 2.23: Use bd_lvm_get_max_lv_size function instead.
+ */
+#define BD_LVM_MAX_LV_SIZE G_GUINT64_CONSTANT (9223372036854775808ULL)
```

Comment:
Would still be nice to see the `8 EiB` value in a comment here.

Reply:
Comment added.